### ISSUE TITLE
Undiscoverable LAN Tessels on OSX

### ISSUE THREAD

**Issue by johnnyman727** [tessel/t2-cli#302]
Sometimes Tessels that are on the same WiFi network are not showing up on `t2 list`. 

You can tell that the device is actually being advertised by running:
```
dns-sd -B _tessel._tcp .
```
You should see something like:
```
➜  dns-sd -B _tessel._tcp .
Browsing for _tessel._tcp
DATE: ---Sun 13 Sep 2015---
19:05:43.886  ...STARTING...
Timestamp     A/R    Flags  if Domain               Service Type         Instance Name
19:05:43.886  Add        2   4 local.               _tessel._tcp.        Tessel-02A337692559
```

So the Tessel is found by your computer but not showing up in `mdns-js` for some reason...

The temporary workaround is to run:

```
sudo killall -HUP mDNSResponder
```
Which I believe clears OSX's mDNS cache...

Then `t2 list` will display your T2 again.

I believe this is mostly an issue with the underlying mdns lib.

**Comment by wprater** [comment]
Im getting this often now.. when I have USB and LAN for the same Tessel

**Comment by johnnyman727** [comment]
@wprater does the workaround immediately clear it up for you?

**Comment by wprater** [comment]
yes, after I run that it will show up.  but then the issue came back
immediately.  Ive plugged the Tessel into the wall for now.  wish I had
more time to track it down...

**Comment by johnnyman727** [comment]
My guess is that it's an underlying issue with the `mdns` library we're using because OSX can find the device just fine with `dns-sd -B _tessel._tcp .`

**Comment by hoegertn** [comment]
Anything new on this issue? I have the same problem and `sudo killall -HUP mDNSResponder` does not help

**Comment by mikeLspohn** [comment]
I'm having the same issue and that's not helping me either. Any word on this?

**Comment by Frijol** [comment]
Checking in on this issue, it looks like we have two scenarios with similar symptoms:

1. one where @johnnyman727's workaround temporarily resolves, and
2. one which the workaround does not fix.

Has anyone experienced #1 since September of 2015?

Has anyone experienced #2 since September of 2016?

If not, I will close this issue. If so, has anyone been able to consistently reproduce the errors?
